dist/modules/sass.mjs

```diff
@@ -11,6 +12,18 @@ export default async function (config) {
     let startTime = new Date().getTime()
     for (let task of config.tasks) {
       let destFileName = task.target || 'dist/styles/style.css'
+/*
```

Comment:
is there a reason to leave this commented out code in?

Reply:
Nope, will remove before merging